src/components/ProgressBars/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -116,41 +126,55 @@ const LinearProgressBar: VibeComponent<LinearProgressBarProps, HTMLDivElement> &
               barStyle={ProgressBarStyle.NONE}
               value={baseValue}
               animated={animated}
-              baseClass={baseClassName}
+              type={ProgressBarType.PRIMARY}
               color={color}
               min={min}
               max={max}
               /* eslint-disable-next-line react/no-array-index-key */
-              key={`${baseClassName}_${color}_${i}`}
+              id={`bar_${color}_${i}`}
+              key={`bar_${color}_${i}`}
             />
           ))}
         </>
       );
     }, [min, max, animated, multiValues, multi]);
 
     const renderPercentage = indicateProgress ? (
-      <PercentageLabel forElement="linear-progress-bar" className={`${baseClassName}__label`} value={valuePercentage} />
+      <PercentageLabel forElement="linear-progress-bar" value={valuePercentage} />
     ) : null;
 
     const renderBaseBars = !multi ? (
       <>
         <Bar
           barLabelName={ariaLabel}
           barStyle={barStyle}
-          id="linear-progress-bar"
+          id="bar-secondary"
```

Comment:
not sure we should give ids at all, do we really need it?

Reply:
mostly we are not giving default ids like that - not sure why we need it, it just was here before

Reply:
Replaced with default data-testid